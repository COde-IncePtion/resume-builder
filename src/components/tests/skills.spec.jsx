import React from "react";
import { shallow } from "enzyme/build";
import Skills from "../skills";

it("should render Skills", () => {
  const component = shallow(<Skills skills={["java", "react"]} />);
  expect(component.debug()).toMatchSnapshot();
});
