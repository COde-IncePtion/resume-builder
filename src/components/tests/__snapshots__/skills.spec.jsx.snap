// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Skills 1`] = `
"<div>
  <Title text=\\"SKILLS\\" fontWeight=\\"700\\" fontSize=\\"28px\\" textAlign=\\"left\\" margin=\\"0 0 4px 50px\\" color=\\"black\\" padding=\\"unset\\" fontStyle=\\"unset\\" />
  <SectionLine />
  <div style={{...}}>
    <Cell skillName=\\"java\\" />
    <Cell skillName=\\"react\\" />
  </div>
</div>"
`;
